lib/src/main/kotlin/cz/eman/swagger/codegen/generator/kotlin/KotlinClientCodegen.kt

```diff
@@ -524,6 +556,34 @@ open class KotlinClientCodegen : org.openapitools.codegen.languages.KotlinClient
         }
     }
 
+    /**
+     * Processes options for the Kotlin multiplatform development. It will apply and multiplatform specific opts only
+     * if:
+     * ```
+     * setLibrary("multiplatform")
+     * ```
+     *
+     * @since 2.2.4
+     */
+    private fun processOptsForMultiplatform() {
+        if (MULTIPLATFORM == getLibrary()) {
+            processKotlinxDateTimeLibrary()
+        }
+    }
+
+    private fun processKotlinxDateTimeLibrary() {
```

Comment:
Přemýšlím proč to máš vyseparované do funkce když to volá funkce která má jeden if. Ale budiž.